cmd/check-spelling/data/main.txt

```diff
@@ -78,6 +83,7 @@ teardown
 templating
 timestamp/AB
 tracability
+ttRPC
```

Comment:
btw, you probably want to make this `ttRPC/B` as that will match "ttRPC" and also "ttRPC's". These one-letter rules are defined here:

- https://github.com/kata-containers/tests/blob/master/cmd/check-spelling/data/rules.aff
  ("A" is for handling plurals and "B" is for possession [`'s`]).

Reply:
done